test-browser/tests/gist.js

```diff
@@ -35,8 +37,8 @@ module.exports = {
         browser
           .modalFooterCancelClick()
           .executeScript(`remix.loadgist('${gistid}')`)
-          .switchFile('browser/gists')
-          .switchFile(`browser/gists/${gistid}`)
+        if (runtimeBrowser === 'chrome') browser.switchFile('browser/gists')
```

Comment:
that's quite unexpected that not running `browser.switchFile('browser/gists')` for firefox still work...

Reply:
Yeah, I think it has to do with the mutiple entry of `.sendKeys('*[data-id="terminalCliInput"]', this.api.Keys.ENTER')` in `.executeScript` and how firefox handles it.